src/native/event_loop_group.c

```diff
@@ -49,17 +49,26 @@ static void s_event_loop_group_cleanup_completion_callback(void *user_data) {
     (*jvm)->AttachCurrentThread(jvm, (void **)&temp_env, NULL);
     env = temp_env;
```

Comment:
Should we also check the result of `AttachCurrentThread` against `JNI_OK`, in case an error that is not an no-op occurs?

Reply:
I don't think it gives you an environment unless it succeeds.

Reply:
I looked and I think you are right, it looks like it just returns an error and will keep `env` as null if there is an error.
It should be good then as is.